### DISCUSSION TITLE
How to use the deploy batch script, not working

### DISCUSSION THREAD

**Discussion by riccardospeggiorin-centropaghe** [rustdesk/rustdesk#9894] [Q&A]
Hi, I am trying to use the batch deployment script to set the relay server and public key on install.
So I am using this batch (no the powershell one) https://rustdesk.com/docs/en/self-host/client-deployment/
I encode in base64 this value for the `rustdesk_cfg` variable :
{"host":"rust.example.com","key":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX","api":"http://rust.example.com:21114"}

I run the script, the install finished but no values are set! All the field are empty and rustdesk says:
1) the service is not running
2) after running the service it says to connect to a relay server and not use the public one (like you download the vanilla version)

anyone can help me?
Many thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, please search there first, many guys have asked similar question because you are not the most special though you always think you are. Usually nobody will help you with build or self-hosting here.
